src/components/AccountDashboard.tsx

```diff
@@ -1,21 +1,82 @@
 import * as React from 'react'
 
-import { Transactions } from '@containers/Transactions'
-import { Transfer } from '@containers/Transfer'
-import { WalletInfo } from '@containers/WalletInfo'
+import { CurrencySelector } from '@containers/CurrencySelector'
 
-export const AccountDashboard: React.SFC = () => (
+import * as kagLogo from '@icons/kag-icon.svg'
+import * as kauLogo from '@icons/kau-icon.svg'
+import { RootState } from '@store'
+import { Currency, RootRoutes } from '@types'
+import * as sleeping from 'images/wally-sleeping.svg'
+import { connect } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+
+const mapStateToProps = (state: RootState) => ({
+  currency: state.connections.currentCurrency,
+  balance: state.accounts.accountInfo.balance,
+})
+
+type Props = ReturnType<typeof mapStateToProps>
+
+const AccountDashboardPresentation: React.SFC<Props> = props => (
   <React.Fragment>
```

Comment:
don't think this needs to be wrapped in a fragment, the container already wraps everything else?